android/src/main/res/layout/fingerprint_dialog.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:orientation="vertical" android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:weightSum="1"
+    android:id="@+id/fingerprint_container">
+
+    <!-- https://medium.com/@chrisbanes/appcompat-v23-2-age-of-the-vectors-91cbafa87c88#.btn3al2l2 appCompat works if app is srcCompat 7 activity-->
+    <!-- app:srcCompat="@drawable/ic_fp_40px" -->
+
+    <TextView
+        android:text="TextView"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:id="@+id/fingerprint_description"
+        android:textColor="#000"
+        android:textSize="17dp"
+        android:layout_marginTop="20dp"
+        android:layout_marginLeft="25dp"
+        android:layout_marginRight="25dp"
+        android:gravity="left" />
+
+
+        <LinearLayout
+                android:layout_width="fill_parent"
+                android:layout_height="wrap_content"
+                android:gravity="center"
+                android:orientation="horizontal"
+                android:paddingBottom="30dp"
+                android:paddingRight="20dp"
+                android:paddingTop="28dp" >
+                <ImageView
+                  android:layout_width="49dp"
+                  android:layout_height="49dp"
+                  android:layout_marginLeft="25dp"
+                  android:src="@drawable/ic_fp_40px"
```

Comment:
I would not generate a png for this icon but you can generate the xml with Android Studio. Then will look good anywhere.